src/pages/better-dev-onboarding.tsx

```diff
@@ -0,0 +1,145 @@
+        type: 'Case study',
+        img: {
+            src: 'https://sourcegraphstatic.com/blog/log4j/log4j-blog-thumbnail.png',
+            alt: 'Log4j Log4Shell 0-day blog thumbnail',
+        },
+        href: '/blog/log4j-log4shell-0-day',
+    },
+]
+
+const BetterDeveloperOnboarding: FunctionComponent = () => (
+    <Layout
+        meta={{
+            title: 'Better Developer Onboarding | Sourcegraph',
+            description:
+                'Find your own answers with self-serve onboarding, codebase exploration, and knowledge sharing, without waiting for someone to point you to the relevant code.',
+        }}
+        className="navbar-light"
+        hero={
+            <>
+                <div className={styles.pageHeader}>
+                    <div className="container pb-4">
+                        <div className="row">
+                            <div className="col-lg-7 mb-8 mt-7">
+                                <BackButtonBold href="/use-cases" text="USE CASES" />
+                                <h1 className="display-2 font-weight-bold mb-4">
+                                    Give your team a complete onboarding experience
+                                </h1>
+                                <div className="display-4 font-weight-normal mb-5">
+                                    With self-serve onboarding, codebase exploration, and knowledge sharing, developers
+                                    can find their own answers without waiting for someone to point them to the relevant
+                                    code.
+                                </div>
+                                <div className="d-flex flex-column flex-lg-row pt-1">
+                                    <Link href="/demo" passHref={true}>
+                                        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+                                        <a
+                                            className="btn btn-primary mr-lg-3 mb-lg-0 mb-3 w-md-100"
+                                            title="Request a Demo."
+                                        >
+                                            Request a demo
+                                        </a>
+                                    </Link>
+                                    <Link href="/get-started" passHref={true}>
+                                        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+                                        <a
+                                            className="btn btn-outline-primary w-md-100"
+                                            title="Get started with Sourcegraph."
+                                        >
+                                            Get started
+                                        </a>
```

Comment:
Can these CTA's be adjusted for medium screens and not take up the whole screen width? `home/_Hero.tsx` has a good example of this for progressive sizing. Can you add button tracking here too?
![image](https://user-images.githubusercontent.com/59381432/169869495-6e0c7949-8f5d-41d3-b694-722a2e85578c.png)

Reply:
Thank you for pointing this out. I added tracking and some responsiveness to these buttons here: 229ca92. What do you think?

This pattern is used throughout the Use Case headers: [Sourcegraph](https://sourcegraph.com/search?q=context:global+repo:github.com/sourcegraph/about+className%3D%22d-flex+flex-column+flex-lg-row+pt-1%22&patternType=literal). If what I've added looks alright with you, I will go ahead and update those as part of this PR.

Reply:
Oh yea that would be a great cleanup task for us to do too. Thanks for finding the references, Tim. I like your solution, just a comment to combine classes into the anchor tag to remove excess `div`s

Reply:
Nice! I combined the classes on this page here: 4bf4a0a. When trying to combine the classes on the Use Case pages, I noticed that removing these divs threw off the appearance and responsiveness, so I left them in. Otherwise, the same classes are applied: c8342df